src/view/payment-request.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import { NamedField } from '../component/field';
+import { Header, Title } from '../component/header';
+import Text from '../component/text';
+import { BackButton, CopyButton, SmallButton } from '../component/button';
+import { BalanceLabel } from '../component/label';
+import Card from '../component/card';
+import QRCode from '../component/qrcode';
+import { colors } from '../component/style';
+
+const styles = StyleSheet.create({
+  header: {
+    minWidth: 500,
+    width: '100%',
+    alignSelf: 'center',
+  },
+  content: {
+    alignItems: 'stretch',
+    justifyContent: 'flex-start',
+    overflow: 'auto',
+  },
+  amount: {
+    color: colors.darkText,
+  },
+  amountWrapper: {
+    marginTop: 20,
+    marginBottom: 20,
+  },
+  hiddenItem: {
+    // Workaround to align header items correctly.
+    opacity: 0,
+  },
+  numeral: {
+    lineHeight: 70,
+    fontFamily: 'WorkSans ExtraLight',
+    fontSize: 103,
+    color: colors.darkText,
+  },
+  unit: {
+    lineHeight: 30,
+    fontFamily: 'WorkSans Regular',
+    fontSize: 23,
+    color: colors.darkText,
+  },
+  doneTouchable: {
+    height: 50,
+    width: 100,
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignSelf: 'center',
+    margin: 20,
+  },
+  doneText: {
+    color: colors.purple,
+    fontFamily: 'OpenSans SemiBold',
+    fontSize: 14,
+  },
+});
+
+const PaymentRequest = ({ store }) => (
+  <Background image="purple-gradient-bg">
+    <Header shadow color={colors.purple} style={styles.header}>
+      <BackButton onPress={() => {}} />
+      <Title title="Payment Request" />
+      <View style={styles.hiddenItem}>
+        <BackButton onPress={() => {}} />
+      </View>
+    </Header>
+    <Card style={styles.content}>
+      <View style={styles.amountWrapper}>
+        <BalanceLabel unit={<Text style={styles.unit}>SAT</Text>}>
```

Comment:
I haven't seen child nodes being injected via attributes. Maybe we can think of a cleaner way to do this. What are you trying to achieve?

Reply:
Maybe we can use `adjustsfontsizetofit` somehow?
https://facebook.github.io/react-native/docs/text.html#adjustsfontsizetofit

Reply:
I was trying to figure out how to style the unit only, but realized this was unnecessary. 

Looks like `adjustsFontSizeToFit` isn't supported in react-native-web: https://necolas.github.io/react-native-web/storybook/?selectedKind=Components&selectedStory=Text&full=0&addons=0&stories=1&panelRight=0

One potential problem that remains is that if someone's balance is greater than 999,999 satoshis then the full balance won't fit. I think an acceptable solution could be either to use a smaller font size or to calculate the appropriate font size based on screen widths?